velodyne_pointcloud/src/conversions/convert.cc

```diff
@@ -21,29 +21,36 @@ namespace velodyne_pointcloud
 {
   /** @brief Constructor. */
   Convert::Convert(ros::NodeHandle node, ros::NodeHandle private_nh):
-    data_(new velodyne_rawdata::RawData())
+    data_(new velodyne_rawdata::RawData()),
+    prev_azimuth_(0.0),
+    prev_stamp_(ros::Time())
   {
     data_->setup(private_nh);
 
 
     // advertise output point cloud (before subscribing to input data)
-    output_ =
+    output_pointcloud_ =
       node.advertise<sensor_msgs::PointCloud2>("velodyne_points", 10);
-      
+    
+    // advertise output deskew info
+    output_deskew_info_ =
+      node.advertise<velodyne_msgs::VelodyneDeskewInfo>("velodyne_deskew_info", 10);
```

Comment:
same comment about the queue size.

Reply:
Have reduced it to 2 and works fine when I tested.